hss/src/resthandler.cpp

```diff
@@ -21,9 +21,156 @@ void RestHandler::onRequest(const Pistache::Http::Request& request, Pistache::Ht
    std::cout << request.resource() << std::endl;
    std::cout << request.method() << std::endl;
    std::cout << request.body() << std::endl;
+   static bool needConfig = true;
 
-   if (request.resource() == "/v1/config/imsis")
-   {
+   if (request.resource() == "/v2/config-check") {
+     if (needConfig == true) {
+        response.send(Pistache::Http::Code::Not_Found);
+        needConfig = false;
```

Comment:
should we set needConfig to false after getting config??

Reply:
Since HSS is per subscriber, I am doing it as soon as config requested.  and not when we really receive configuration.